driver/src/main/scala/core/nodeset.scala

```diff
@@ -1,19 +1,34 @@
 package reactivemongo.core.nodeset
 
-import java.util.concurrent.{ Executor, Executors, TimeUnit }
+import java.util.concurrent.{ TimeUnit, Executor, Executors }
 
 import scala.collection.generic.CanBuildFrom
 import scala.collection.immutable.Set
+
 import shaded.netty.util.HashedWheelTimer
 import shaded.netty.buffer.HeapChannelBufferFactory
 import shaded.netty.channel.socket.nio.NioClientSocketChannelFactory
-import shaded.netty.channel.{ Channel, ChannelFuture, ChannelPipeline, Channels }
+import shaded.netty.channel.{
```

Comment:
yeah I know, it's useful for most projects I commit on, I'll disable it so it doesn't happen on reactive mongo